### PR TITLE
Update connection user experience

### PR DESCRIPTION
**Pull request by shanghaikid**
- support disconnect
- support keyboard event "enter"
- save last time milvus ip address
- support connect with ip without port
- clear state on logout

### CODE REVIEW

File: client/src/components/layout/Header.tsx [hunk 1]
```diff
@@ -70,10 +71,13 @@ const Header: FC<HeaderType> = props => {
...
   };
 
-  const handleLogout = () => {
+  const handleLogout = async () => {
     setAddress('');
     setIsAuth(false);
+    await MilvusHttp.closeConnection();
     window.localStorage.removeItem(MILVUS_ADDRESS);
+    // make sure we clear state in all pages
+    navigate(0);
```
Comment (nameczz) [inline client/src/components/layout/Header.tsx:80]: what's this for?

File: client/src/utils/Format.ts [hunk 2]
```diff
@@ -141,8 +142,11 @@ export const getCreateFieldType = (config: Field): CreateFieldType => {
 };
 
 // Trim the address
-export const formatAddress = (address: string): string =>
-  address.trim().replace(/(http|https):\/\//, '');
+export const formatAddress = (address: string): string => {
+  // remove http or https prefix from address
+  const ip = address.replace(/(http|https):\/\//, '');
+  return ip.includes(':') ? ip : `${address}:${DEFAULT_MILVUS_PORT}`;
```
Comment (nameczz) [inline client/src/utils/Format.ts:148]: `${address}:${DEFAULT_MILVUS_PORT} ` , address -> ip

File: server/src/milvus/milvus.controller.ts [hunk 3]
```diff
@@ -97,4 +94,14 @@ export class MilvusController {
     };
     res.send(data);
   }
+
+  closeConnection(req: Request, res: Response, next: NextFunction) {
+    try {
+      const result = this.milvusService.closeConnection();
+      res.send({ result });
+    } catch (error) {
+      console.log(error);
```
Comment (nameczz) [inline server/src/milvus/milvus.controller.ts:103]: do we need console here?

**Review by nameczz — CHANGES_REQUESTED** [review]

File: client/src/components/layout/Header.tsx (hunk 1 above)
Comment (shanghaikid) [reply client/src/components/layout/Header.tsx:80]: Reload the page.

**Pull request merged**